src/site.rs

```diff
@@ -466,6 +466,7 @@ fn handle_404(
     let mut content = Content {
         html: String::from("Page not found :/"),
         title: String::from("Page not found"),
+        description: Some(String::from("")),
```

Comment:
We can set to None here, no need to build an empty string.
```suggestion
        description: None,
```